jsettlers.logic/src/main/java/jsettlers/logic/movable/Movable.java

```diff
@@ -53,11 +54,10 @@
  * @author Andreas Eberle
  *
  */
-public final class Movable implements IScheduledTimerable, IPathCalculatable, IDebugable, Serializable, IViewDistancable, IGuiMovable,
-		IAttackableMovable {
+public final class Movable implements ILogicMovable {
 	private static final long serialVersionUID = 2472076796407425256L;
-	private static final HashMap<Integer, Movable> movablesByID = new HashMap<Integer, Movable>();
-	private static final ConcurrentLinkedQueue<Movable> allMovables = new ConcurrentLinkedQueue<Movable>();
+	private static final HashMap<Integer, ILogicMovable> movablesByID = new HashMap<Integer, ILogicMovable>();
```

Comment:
As the name suggests these fields contain all movables which are not exclusively used by the movable class. Other actors access them with `Movable.getMovableByID()`. The movables with the new architecture under the hood should be present in these fields, too. They are not a subclass of Movable, therefore `ILogicMovable` is necessary here. 
Later I decided to move the static fields into a separate `MovableDataManager` class in my commit b1f911c5c4b6c7aca27c22d4365e29fd5b49d83d